src/WebJobs.Script/Binding/HttpBinding.cs

```diff
@@ -98,16 +106,24 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes(Type para
                     Content = new StringContent((string)content)
```

Comment:
I logged https://github.com/Azure/azure-webjobs-sdk-script/issues/882 for us to clean this up